components/cart/CartLayout.module.scss

```diff
@@ -0,0 +1,29 @@
+.cartLayout {
+  max-width: 175rem;
+  margin: auto;
+  padding: 4rem 2rem;
+
+  &__container {
```

Comment:
Why have different containers for different elements? why not one global container?

Reply:
Okk